cfgov/unprocessed/apps/cfpb-chart-builder/js/charts/GeoMap.js

```diff
@@ -141,8 +141,6 @@ class GeoMap {
     el.classList.add('cfpb-chart');
     el.setAttribute('data-chart-type', 'geo-map');
 
-    // TODO: remove when gulp build config is updated to handle spread operator.
-    // eslint-disable-next-line prefer-object-spread
     this.chart = Highcharts.mapChart(el, Object.assign({}, this.chartOptions));
```

Comment:
`prefer-object-spread` isn't set by default in `@eslint.js.configs.recommended`. However, we could enable it with `prefer-object-spread: [error]` in the rules and it'll auto-update this to the spread operator.